host-configs/lc-builds/blueos/xl_X.cmake

```diff
@@ -7,10 +7,10 @@
 
```

Comment:
Isn't this file the same as the xl_2020_X file?

Reply:
Removed, it's not being used anyway.